### PR TITLE
Rhosak UI new codebase

### PR DESCRIPTION
**Pull request by mmusil**
PR is connecter to https://gitlab.cee.redhat.com/mk-ci-cd/mas-ci/-/merge_requests/772

### CODE REVIEW

File: lib/navigation.ts [hunk 1]
```diff
@@ -35,7 +36,12 @@ export const navigateToProductList = async function (page: Page, product: string
 
 // Navigates to list of Streams for Apache Kafka instances
 export const navigateToKafkaList = async function (page: Page) {
-  await navigateToProductList(page, 'Streams for Apache Kafka', 'Kafka Instances');
+  if (process.env.ENV_TYPE == 'stage-new-ui') {
+    await page.goto(config.startingPage + '/application-services/new-streams');
```
Comment (Frawless) [inline lib/navigation.ts:40]: shouldn't this be set in startingPage env var ? And just via `ENV_TYPE` change selectors?

Comment (mmusil) [reply lib/navigation.ts:40]: It broke the version checks in pipeline stages, since it uses the same env variable.

Comment (Frawless) [reply lib/navigation.ts:40]: so in general the problem is, that UI is the same for everything except streams, right?

**Review by Frawless — DISMISSED** [review]

**Review by kornys — APPROVED** [review]

**Review by Frawless — DISMISSED** [review]

File: lib/pom/abstractPage.ts [hunk 2]
```diff
@@ -34,6 +34,10 @@ export abstract class AbstractPage {
     this.confirmDeleteField = page.getByLabel('Type DELETE to confirm:');
     this.saveButton = page.getByRole('button').filter({ hasText: 'Save' });
 
+    if (process.env.ENV_TYPE == 'stage-new-ui') {
```
Comment (kornys) [inline lib/pom/abstractPage.ts:37]: can we load this env into config and use that?

File: lib/pom/streams/instance/consumerGroups.ts [hunk 3]
```diff
@@ -2,20 +2,21 @@ import { expect, Locator, Page } from '@playwright/test';
...
-  readonly consumerGroupsMenuButton: Locator;
   readonly consumerGroupIdButton: Locator;
   readonly consumerGroupHeading: Locator;
 
   constructor(page: Page, instanceName: string) {
     super(page, instanceName);
-    this.consumerGroupsMenuButton = page.locator('button', { hasText: 'Consumer groups' });
     this.consumerGroupIdButton = page.locator('button', { hasText: 'Consumer group ID' });
     this.consumerGroupHeading = page.locator('h2', { hasText: 'No consumer groups' });
+    if (process.env.ENV_TYPE == 'stage-new-ui') {
```
Comment (kornys) [inline lib/pom/streams/instance/consumerGroups.ts:12]: same as above

File: lib/pom/streams/kafkaInstanceList.ts [hunk 4]
```diff
@@ -28,6 +28,10 @@ export class KafkaInstanceListPage extends AbstractPage {
     this.connectionButton = page.locator('button', { hasText: 'Connection' });
     this.bootstrapField = page.locator('[aria-label="Bootstrap server"]');
     this.closeDrawerButton = page.locator('[aria-label="Close drawer panel"]');
+
+    if (process.env.ENV_TYPE == 'stage-new-ui') {
```
Comment (kornys) [inline lib/pom/streams/kafkaInstanceList.ts:32]: same as above

File: tests/rhosak/kas_with_instance.spec.ts [hunk 5]
```diff
@@ -50,14 +50,16 @@ test.afterEach(async ({ page }) => {
 
   await kafkaInstancesPage.gotoThroughMenu();
   await kafkaInstancePage.gotoThroughMenu();
-  await topicPage.gotoThroughMenu();
+  if (process.env.ENV_TYPE != 'stage-new-ui') {
```
Comment (kornys) [inline tests/rhosak/kas_with_instance.spec.ts:53]: same a sabove

**Review by kornys — CHANGES_REQUESTED** [review]

**Review by kornys — DISMISSED** [review]

**Review by Frawless — APPROVED** [review]

**Review by Frawless — APPROVED** [review]

**Review by kornys — APPROVED** [review]

**Pull request merged**